src/views/virtualmachines/actions/VirtualMachineActionFactory.tsx

```diff
@@ -262,6 +263,22 @@ export const VirtualMachineActionFactory = {
       label: t('Move to folder'),
     };
   },
+  openConsole: (vm: V1VirtualMachine): ActionDropdownItemType => {
+    const vmCluster = getCluster(vm);
+    const vmNamespace = getNamespace(vm);
+    const vmName = getName(vm);
+
+    return {
+      accessReview: asAccessReview(VirtualMachineModel, vm, 'get'),
```

Comment:
This will fire an unecessary call for a 'get' permission. But if u get the vm, u have get access